website/project/model.py

```diff
@@ -118,6 +120,31 @@ def requires_consent(self):
     def has_files(self):
         return self._config.get('hasFiles', False)
 
+    @property
+    def absolute_api_v2_url(self):
+        path = '/metaschemas/{}/'.format(self._id)
+        return api_v2_url(path)
+
+    # used by django and DRF
+    def get_absolute_url(self):
+        return self.absolute_api_v2_url
+
+    def validate_metadata(self, *args, **kwargs):
```

Comment:
I agree that it can go in a follow-up PR. Even though I don't have any tests specifically for this new "validate_metadata" method that has been moved on the model there are still dozens of APIv2 tests for creating and updating drafts, and creating drafts from registrations, all which are testing that this functionality is working.